src/style.css

```diff
@@ -0,0 +1,129 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.page {
+  box-sizing: border-box;
+  width: 100%;
+  border-radius: 0;
+  margin: 0 auto;
+  background: #fff;
+  position: relative;
+}
+
+.bigForm {
+  height: 70px;
+  margin: 20px 0;
+}
+
+.bigField {
+  position: relative;
+}
+
+.bigBar {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+  background: #fff;
+  font-family: inherit;
+  line-height: 22px;
+  font-size: 16px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding: 23px 0 23px 62px;
+}
+
+.bigBar::placeholder {
+  font-family: inherit;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.bigBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.bigBar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  font-weight: 900;
+  text-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  outline: none;
+}
+
+.smallField {
+  position: relative;
+  height: 42px;
+  background: #fff;
+}
+
+.smallBar {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding: 10px 0 10px 33px;
+  font-family: inherit;
+  font-size: 14px;
+  font-weight: 300;
+  line-height: 19px;
+}
+
+.smallBar::placeholder {
+  font-family: inherit;
+  font-size: 14px;
+  font-weight: 300;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.smallBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.smallBar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: 900;
+  text-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  outline: none;
+}
+
+.bigButton {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  padding-top: 26px;
+}
+
+.smallButton {
+  position: absolute;
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  padding-top: 15px;
```

Comment:
check the previous comments
<img width="936" alt="image" src="https://user-images.githubusercontent.com/102411130/212848901-0c090165-d81b-4e53-a0f6-5533d1527857.png">
<img width="477" alt="image" src="https://user-images.githubusercontent.com/102411130/212848937-4950c6a8-e034-44eb-a953-935345837fcf.png">